android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -697,7 +697,7 @@ public void onPictureTaken(byte[] data, Camera camera) {
                         isPictureCaptureInProgress.set(false);
 
                         mOrientation = Constants.ORIENTATION_AUTO;
-                        mCallback.onPictureTaken(data, displayOrientationToOrientationEnum(mDeviceOrientation));
+                        mCallback.onPictureTaken(data, displayOrientationToOrientationEnum(mDeviceOrientation), 0);
```

Comment:
Camera1 implementation doesn't have the orientation value? Why the hard-coded 0? I believe Camera1 has `getCameraOrientation` to get it's currently selected camera orientation.

Reply:
Thank you. That was nice catch. I didn't recognize there was getCameraOrientation function. I have just checked there is no mCameraOrientation member variable, and I thought which is why the cameramanager doesn't suppout <21 android versions. Anyway,  I pushed new commit. Pls check it.